playwright/e2e/crypto/utils.ts

```diff
@@ -22,6 +23,47 @@ import { Client } from "../../pages/client";
 import { ElementAppPage } from "../../pages/ElementAppPage";
 import { Bot } from "../../pages/bot";
 
+/**
+ * Create a new device for the Alice user.
```

Comment:
As far as I can tell, this does *not* create a new device for any user, Alice or otherwise. Rather, it takes the credentials for an *existing* device (for a user who may or may not be called "Alice") and wraps them up with a `Bot` class.

You'd have to do an actual login to create a new device.